rust/cx-sdk/src/lib.rs

```diff
@@ -31,6 +34,19 @@ mod util;
 
 const ENV_CORALOGIX_REGION: &str = "CORALOGIX_REGION";
 
+const AUTHORIZATION_HEADER_NAME: &str = "authorization";
+
+pub(crate) const SDK_VERSION_HEADER_NAME: &str = "cx-sdk-version";
+pub(crate) const SDK_LANGUAGE_HEADER_NAME: &str = "cx-sdk-language";
+pub(crate) const SDK_RUSTC_VERSION_HEADER_NAME: &str = "cx-sdk-rustc-version";
+pub(crate) const SDK_CORRELATION_ID_HEADER_NAME: &str = "cx-sdk-correlation-id";
+
+/// The Rust compiler version.
+pub(crate) const RUSTC_VERSION: &str = env!("RUSTC_VERSION");
+
+/// The SDK version.
+pub(crate) const SDK_VERSION: &str = env!("CARGO_PKG_VERSION");
```

Comment:
This can actually be `pub` so no `pub(crate)`

Reply:
Done